mm2src/coins/eth/eth_swap_v2.rs

```diff
@@ -18,6 +29,26 @@ struct TakerFundingArgs {
     payment_time_lock: u32,
 }
 
+struct TakerRefundSecretArgs {
+    dex_fee: U256,
+    payment_amount: U256,
+    maker_address: Address,
+    taker_secret: [u8; 32],
+    maker_secret_hash: [u8; 32],
+    payment_time_lock: u32,
+    token_address: Address,
+}
```

Comment:
you can delete this structure and reuse `TakerRefundArgs` everywhere

Reply:
Done here https://github.com/KomodoPlatform/komodo-defi-framework/pull/2169/commits/0bae344289e12555df88b3b9969500491cce5887
I added `taker_secret` field to not to confuse it with `taker_secret_hash`